scrum/Card Size Power-up Module.md

```diff
@@ -0,0 +1,31 @@
+# Purpose
+
+The [card size power-up module](https://trello.com/b/mPW9C7y6/sprint-now/power-up/5cd476e1efce1d2e0cbe53a8) was installed onto both the "SPRINT-now" board and the "READY FOR SPRINT" section in the "DISCOVERY" board.
+
+The purpose of installing the power-up is to help with the calculation of how many cards are in various columns on a Trello board and seeing how many points of work are allocated to a particular assignee and label.
+
+Another future purpose is combining this module with another one that can access the card size of cards and make charts and dashboard.
+
+# Required by
+
+End-of-Sprint calculations
+
+(In the future) [Dashboards by Screenful power-up module](https://trello.com/b/mPW9C7y6/sprint-now/power-up/570262ea1100fa611d7e200a)
+
+# Maintenance Requirements
+
+There are a few problems currently that should be kept in mind when working with this module.
+
+Because there are only three buckets to determine the size of a card, there is no built-in way to keep track of retro points for a given card.
+Could be solved with a Trello custom field that holds the amassed retro points. (Alex, Matt, and Duncan are leaning towards this solution)
+This means that the total amount spent on a card is still there in case for reference and could be pulled by a burndown chart for additional Sprint analysis.
```

Comment:
just a general comment - I'm interested in burndown... but even if we have total amount spent on card that we could accurately pull... how do we get burndown from this... is the idea here that we would use this in conjunction that engineers would 'update' SP 'remaining' at the end of each day on their cards?